app/upgrades/v15/upgrades.go

```diff
@@ -350,3 +356,94 @@ func SetMinInitialDepositRatio(ctx sdk.Context, gk govkeeper.Keeper) error {
 
 	return nil
 }
+
+/*
+The following is a list of the discrepancies that were found in the IBC transfer escrow accounts.
+
+Discrepancy #1:
+- Counterparty Chain ID: osmosis-1
+- Escrow Account Address: cosmos1x54ltnyg88k0ejmk8ytwrhd3ltm84xehrnlslf
+- Asset Base Denom: FX
+- Asset IBC Denom: ibc/4925E6ABA571A44D2BE0286D2D29AF42A294D0FF2BB16490149A1B26EAD33729
+- Escrow Balance: 8859960534331100342
+- Counterparty Total Supply: 8899960534331100342ibc/EBBE6553941A1F0111A9163F885F7665417467FB630D68F5D4F15425C1E64FDE
+- Missing amount in Escrow Account: 40000000000000000
+
+Discrepancy #2:
+- Counterparty Chain ID: osmosis-1
+- Escrow Account Address: cosmos1x54ltnyg88k0ejmk8ytwrhd3ltm84xehrnlslf
```

Comment:
Is this supposed to be the same account address as for Discrepancy #1?

Reply:
This escrow account address has two discrepancies for different coin denoms.